backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/ServerInfoSynchronizer.java

```diff
@@ -41,6 +43,20 @@ public void synchronize(ServerApi serverApi, SonarLintCancelMonitor cancelMonito
     var serverStatus = serverApi.system().getStatus(cancelMonitor);
     var serverVersionAndStatusChecker = new ServerVersionAndStatusChecker(serverApi);
     serverVersionAndStatusChecker.checkVersionAndStatus(cancelMonitor);
-    storage.serverInfo().store(serverStatus);
+    var multiQualityMode = getMultiQualityMode(serverApi.settings(), cancelMonitor);
+    storage.serverInfo().store(serverStatus, multiQualityMode);
+  }
+
+  private static MultiQualityMode getMultiQualityMode(SettingsApi settingsApi, SonarLintCancelMonitor cancelMonitor) {
```

Comment:
Could we already check the server version here and set the value only to either `MQR` or `STANDARD`? This way all the logic can be contained in one method, and there is no need for the `MultiQualityMode.DEFAULT`

Reply:
So in this case we only handle MQR or Standard, which means we could probably get rid of the Enum and just keep a boolean isMQRMode basically